### DISCUSSION TITLE
Distinguish Chrome profiles (e.g. by Icon)

### DISCUSSION THREAD

**Discussion by bugsnoffer** [lawand-dot-io/taskbar#22] [Feature requests]
One great Windows feature is the ability to distinguish between Chrome Profiles by Icon on the taskbar (it even groups by profile)
Just the icon would be useful as a start

**Comment by lawand-dot-io** [comment]
Hello. I will have to look into this. I will only be able to implement this feature if Google Chrome was allowing access to the chrome profile of each window :)

**Comment by lawand-dot-io** [comment]
For reference: apply this feature to other browsers too. Also, consider adding an option where specific windows can be tagged by a color or something similar.

**Comment by maik-intellicoach** [comment]
Agree, please allow this for any browser. Even better, allow manual change of icon to be replaced with picture!

**Comment by meirion** [comment]
Oh wow this would be good. Edge browser and a few others are based on Chrome (Chromium?).

These browser also allow picking a custom colour for each profile which tints the browser UI. Having this tint also on the Taskbar window buttons would be great.

<img width="502" alt="image" src="https://github.com/user-attachments/assets/42106860-325c-4a7a-a745-709611f1d78f" />

**Reply by lawand-dot-io** [reply]
I am not sure the browsers will give me access to such features, so I am considering adding an option to give individual windows custom titles and/or icons. Maybe also custom tints or so. Let's see.
